### ISSUE TITLE
Selector rules should ignore Less control structures

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1342]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

In Scss we get this feature as `at-rule`, but in less it is `rule`, now all `selector-*` rules works incorrect with construction(`for`, `each` and etc). Example (`"selector-no-combinator": true`):
```less
.for(@n: 1) when (@n <= 10) { 
  .n-@{n} { content: %(\"n: %d\", 1 + 1); } 
  .for(@n + 1); 
}
```
Error in each space inside `.for(@n: 1) when (@n <= 10)`.
/cc @davidtheclark @jeddy3

**Comment by evilebottnawi** [comment]
Found solution, coming soon PR

**Comment by davidtheclark** [comment]
Closed by #1344.
